main.go

```diff
@@ -226,16 +230,20 @@ func main() {
 
 	//
 	// Fastlane session
-	fmt.Println()
-	log.Infof("Ensure cookies for Apple Developer Portal")
-
-	fs, errors := devportalservice.SessionData()
-	if errors != nil {
-		log.Warnf("Failed to activate the Bitrise Apple Developer Portal connection: %s\nRead more: https://devcenter.bitrise.io/getting-started/connecting-apple-dev-account/ \nerrors:")
-		for _, err := range errors {
-			log.Errorf("%s\n", err)
+	fs, err := devportalservice.SessionData()
+	if err != nil {
+		if networkErr, ok := err.(devportalservice.NetworkError); ok && networkErr.Status == http.StatusNotFound {
+			log.Debugf("")
+			log.Debugf("Connected Apple Developer Portal Account not found")
```

Comment:
Could you please add a description of why the "Connected Apple Developer Portal Account" is needed?

Something like:
"Connected Apple Developer Portal Account not found. the Apple Developer Portal Account is used for iOS projects when the Fastlane communicates with the Apple Developer Portal.
For none iOS projects, it's not needed and you can ignore the warning."